Dockerfile.in

```diff
@@ -1,11 +1,16 @@
 FROM alpine/helm:VERSION
 ENV HOME /tmp
 
+#Install yq and jq dependencies
+RUN apk update && \
+    apk add py-pip jq && \
+    pip install yq
```

Comment:
`--no-cache` is used below, so should probably use it here too.

Reply:
done, thanks !!